packages/oui-stepper/src/stepper.controller.js

```diff
@@ -21,6 +24,11 @@ export default class {
                 .removeAttr("name")
                 .addClass("oui-stepper")
         );
+
+        this.$scope.$watch(
```

Comment:
You can avoid using `$scope.$watch` by using `get`/`set` of es6 :-)

Reply:
I tried to but I get `RangeError: Maximum call stack size exceeded` (I suppose due to the fact that `currentIndex` is needed to be set elsewhere)